### DISCUSSION TITLE
How to hosting in AWS

### DISCUSSION THREAD

**Discussion by AmitayuZeron** [upptime/upptime#994] [Q&A] [answered]
Hi Upptime Team,

I have successfully configured Upptime on GitHub. Could you please guide me on how to host it on AWS Cloud?

Thank you!

**Comment by AnandChowdhary** [comment, marked as answer]
It's not possible to host Upptime elsewhere since it's based on GitHub Actions, Issues, and Pages. It's Upptime 🤝 GitHub forever. :)

**Comment by AmitayuZeron** [comment]
Thank you @AnandChowdhary for your support. how can I remove Past Incident's history data?

**Reply by AnandChowdhary** [reply]
You can delete issues from GitHub and they should disappear from your status page too: https://docs.github.com/en/issues/tracking-your-work-with-issues/deleting-an-issue
